### PR TITLE
index.js

### PR DESCRIPTION
**Pull request by Vexcon1**
What is added: 
-getrank command
Where you can get your rank name in the group!

### CODE REVIEW

**Comment by MuddyOyester3** [comment]
I think it is an okay idea, but it can be better, Maybe get info, and it shows there rank, there profile picture, there Player Id, their username, and when their account was created, That would make some better use out of it.

File: index.js [hunk 1]
```diff
@@ -541,8 +541,29 @@ client.on("message", async message => {
            }});
         });
     }
-
+if(command === 'getrank') {
```
Comment (MuddyOyester3) [inline index.js]: maybe do `getinfo`

**Comment by MuddyOyester3** [comment]
In my opinion, I would not accept it at the moment, but if you do the things I suggested in the last comment, Then I would probably accept this.

**Comment by yogurtsyum** [comment]
This has actually been suggested before. It’s a current feature on Hyper and will be enhanced even more when verification comes to Hyper. This would currently not be very beneficial to qbot as I’d already have to make a more enhanced one when 5.0/verification comes to both Hyper and qbot. Plus the formatting is a bit weird. Thanks for your pull request though.

File: index.js (hunk 1 above)
Comment (Vexcon1) [reply index.js]: That works too.

**Pull request closed without merging**